functions/schema.html

```diff
@@ -0,0 +1,26 @@
+/// NOT A FUNCTIONAL FILE, JUST NOTES //Default Welcome Intent HCPSignup
```

Comment:
come again?

Reply:
removed - notes for myself about chat flow that I didn't clean up.